.travis.yml

```diff
@@ -70,10 +69,6 @@ install:
         # Use rst2man from python3
         - sudo ln -s /usr/share/docutils/scripts/python3/rst2man /usr/bin/rst2man.py
 
-        # Install clear linux certificate
-        - wget https://download.clearlinux.org/releases/13010/clear/Swupd_Root.pem
-        - sudo install -D -m0644 Swupd_Root.pem /usr/share/clear/update-ca/Swupd_Root.pem
```

Comment:
why we don't need this anymore?

Reply:
I think we don't need that for a long time. I don't even know what's it doing here.
We don't use the Swupd_Root from clear in any tests we run and we don't want to use. So it's better not to install it. Probably it's a leftover from the tests that were used in the past

Reply:
Looks for all the tests we now use Swupd_Root.pem from test/functiona/Swupd_Root.pem

Reply:
thanks for the explanation